parser.go

```diff
@@ -344,6 +344,8 @@ func climbStuff(feature string) {
 		fmt.Println("From on top of the dining room table you can get a better look at the candelabra.")
 		curRoom.Items["candelabra"].Discovered = true
 		fmt.Println("There's wax everywhere but it looks like there might still be a bit of candle left. Is that a candle? Look closer")
+	} else if feature == "down" {
+		climbedUp = false
```

Comment:
We should inform the player their state has changed, ie: 
`fmt.Println("You climb back down to the ground before you get dizzy!")`

Reply:
Done!